tests/test_ctl.py

```diff
@@ -246,7 +246,7 @@ def test_reload(self, mock_get_dcs, mock_post):
     @patch('patroni.ctl.get_dcs')
     def test_restart_reinit(self, mock_get_dcs):
         mock_get_dcs.return_value.get_cluster = get_cluster_initialized_with_leader
-        result = self.runner.invoke(ctl, ['restart', 'alpha'], input='y\n\nnow')
+        result = self.runner.invoke(ctl, ['restart', 'alpha'], input='now\ny\n')
```

Comment:
Since you changed the ordering of interactive questions you also have to fix the `input` for remaining tests